src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlCompositeMemberTranslator.cs

```diff
@@ -26,8 +26,16 @@ public class NpgsqlCompositeMemberTranslator : RelationalCompositeMemberTranslat
             [NotNull] INpgsqlOptions npgsqlOptions)
             : base(dependencies)
         {
-            // ReSharper disable once VirtualMemberCallInConstructor
+            var versionDependentTranslators = new IMemberTranslator[]
+            {
+                new NpgsqlArrayMemberTranslator(npgsqlOptions.PostgresVersion)
+            };
+
+            // ReSharper disable once DoNotCallOverridableMethodsInConstructor
             AddTranslators(MemberTranslators);
+
+            // ReSharper disable once DoNotCallOverridableMethodsInConstructor
```

Comment:
nit: remove space and Resharper comment to put `AddTranslators()` lines together. Can use a Resharper region disable for the warning (begin/end).